### ISSUE TITLE
Access violation when exiting

### ISSUE THREAD

**Issue by thany** [transmission-remote-gui/transgui#1341]
When [#1339](adding a magnet freezes) and I try to exit the program by going Torrent->Exit, this pops up:

![image](https://user-images.githubusercontent.com/152227/91980695-f483b080-ed27-11ea-8c91-451dbbf92c92.png)

I really hope those addresses mean something to someone.

Windows 10
Transmission GUI 5.18.0
Transmission daemon 3.00

**Comment by Cris--** [comment]
Geting this also if i add much torrents in short time.

**Comment by JamesEdge231** [comment]
I get this constantly, at least 50% chance of it happening if I add a torrent. 

Windows 10 2H20 (but the error/bug has been around a while) 
Transmission GUI 5.18.0
Transmission daemon 3.00

**Comment by guytanium** [comment]
Also getting this regularly when adding torrents, changing directory, or queue order.

Windows 10 19041.572
Transmission GUI 5.18.0
Transmission daemon 3.00

**Comment by Merwenus** [comment]
Same error for me too for a long time, freezing while adding a torrent file from Firefox and crashing (this error message) when force close TransGui.
Windows 10 2H20
Transmission latest from Docker
TransGUI 5.18

**Comment by Tharn** [comment]
Yeah it really happens a lot lately, like 50:50 chance to get an Access Violation just adding a single torrent. The client hangs and I get this error when I force exit.

**Comment by thany** [comment]
I've switched to [Tremotesf](https://github.com/equeim/tremotesf2).
